packages/qwik-city/utils/fs.ts

```diff
@@ -33,7 +33,9 @@ export function getPathnameFromDirPath(opts: NormalizedPluginOptions, dirPath: s
     // remove grouped layout segments
     .filter((segment) => !isGroupedLayoutName(segment))
     .join('/');
-
+  if (pathname === '') {
```

Comment:
we are always applying this change and not only when `opts.trailingSlash` is false but I think this is ok

Reply:
I know. But basepathname defaults to a “/” in the project